wurst/closures/ClosureKeyPresses.wurst

```diff
@@ -0,0 +1,106 @@
+package ClosureKeyPresses
+import HashMap
+
+/*  
+    Example usage:
+        onKeyPress(OSKEY_A) -> 
+            print("A player pressed on the A key.")
+
+        onKeyPress(OSKEY_B, OSKEY_META.SHIFT) -> 
+            print("A player pressed on the B key while holding a SHIFT key.")
+
+        onKeyPress(OSKEY_C, false) -> 
+            print("A player released the C key.")
+
+        onKeyPress(players[0], OSKEY_D, OSKEY_META.ALT, false) -> 
+            print("Player 0 released the D key while holding an ALT key.")
+
+    Important:
+        * When the metakey NONE is passed or when no metaKey is given like in `onKeyPress(OSKEY_A, () -> print("A"))`,
```

Comment:
Okay, as discussed in chat, I think as part of a combination that should be the correct behavior,
but this leaves me 2 questions, does the R-release event fire if I:
1. press R
2. press meta key
3. release R

and 
1. press R
2. press meta key
3. release meta key
4. release R

?

Reply:
In the first case it doesn't fire.
In the second it does.

Reply:
I guess that's fine?